libs/wingsdk/src/std/resource.ts

```diff
@@ -14,6 +14,11 @@ export interface IInflightHost extends IResource {
    * Adds an environment variable to the host.
    */
   addEnvironment(name: string, value: string): void;
+
+  /**
+   * Returns the set of environment variables for this host.
+   */
+  get env(): Record<string, string>;
```

Comment:
This is nice :)  I'm just not sure if any resource can hold an `env`, maybe you can add it only to the container for now?

Reply:
Let's try to avoid adding direct `env` access in `IInflightHost` if it's possible for now. Accessing the environment variables be error prone since when you write `let x = host.env();` in userland, it's possible someone else will call `addEnvironment(...)` on the host afterwards, but then `x` will not reflect the new environment variable, leading to hard to find bugs.